cms/models.py

```diff
@@ -221,6 +226,82 @@ def get_context(self, request, *args, **kwargs):
         )
 
 
+class CertificateIndexPage(RoutablePageMixin, Page):
+    """
+    Certificate index page placeholder that handles routes for serving
+    certificates given by UUID
+    """
+
+    parent_page_types = ["HomePage"]
+    subpage_types = []
+
+    slug = CERTIFICATE_INDEX_SLUG
+
+    @classmethod
+    def can_create_at(cls, parent):
+        """
+        You can only create one of these pages under the home page.
+        The parent is limited via the `parent_page_type` list.
```

Comment:
Maybe later, I will suggest to keep the `HomePage` as central part and should keep the other pages in it instead of at the root level. 
Currently, `HomePage` contains the `CertificateIndexPage` while `Signatories` are the root level and `Courses/Programs` --- > `CertificatePage`

Reply:
That's plausible, however on the flip side we have the certificates right next to the course/program it belongs to. The index page for certificates is just a placeholder for routing so it all boils down to what the marketing team will find easier.

Reply:
I don't think I understand what `CertificateIndexPage` is. Can you explain and/or point to a screenshot?

Reply:
@pdpinch the `CertificateIndexPage` is not a page object that will ever be rendered and is there only to handle the routing for the course and program certificate URLs. There is no UI for it. Previously I had added the routing logic to `HomePage` and as per @asadiqbal08 suggestion it was rather non-intuitive to handle that over there, so I simply extracted it out into a `CertificateIndexPage` class that sits at the `/certificate` route and handles the `/certificate/<uuid>/` and `/certificate/program/<uuid>/` URLs. If you try to access `/certificate/` (the index page itself) it will raise a 404.